lib/mayaHydra/hydraExtensions/mixedUtils.cpp

```diff
@@ -78,6 +78,11 @@ SdfPath DagPathToSdfPath(
     const bool      stripNamespaces)
 {
     std::string name = dagPath.fullPathName().asChar();
+    if ( name.empty() ) {
+        MFnDependencyNode dep(dagPath.node());
+        TF_WARN("Empty fullpath name for DAG object : %s of type : %s", dep.name().asChar(), dep.typeName().asChar());
```

Comment:
Do we know when this happens?  Is this for non-Dag nodes?  Does this indicate missing functionality, missing logic, or a bug?

Reply:
I had this with Bifrost nodes and also when selecting the default material set. But the problem is that wer get dag nodes with no type and no name, we should try detect useful dag nodes that need to be translated as some may not need to be treated.